### ISSUE TITLE
Should bold be enabled in headings?

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/editor-recommendations#54]
Emphasising anything in headings doesn't make much sense anyway. Plus, very often, the style is simply invisible.

**Comment by mlewand** [comment]
In terms of presentation, by default, all the headers have the same font-weight as `strong` / `b` element.

In terms of structure, it's a valid HTML to have `strong` / `b` in a header.

In terms of semantics "on paper" it might sound as if `strong` should be allowed in header as anywhere else. However headers should **always** be as short and concise as possible, thus not require any emphasize on it's part. So I'd agree that it has no use in headers.

So to sum it up, I think in reality there's no real reason to allow `strong` in headers. Still, I'd go saying that editor __may__ enable bold in headers, leaving it up to editor implementator.

**Comment by Comandeer** [comment]
Some context: https://github.com/whatwg/html/issues/3597

**Comment by duracell80** [comment]
Yes headers should be able to be bolded. We should not be making decisions on the users behalf, there might be a need to emphasize a heading beyond its text size.

Additionally ... Headers (all elements) should be "style-able" by the user using classes (or templates/themes). The editor could then much like a powerpoint presentation ... be built based on templates where the H1's, bullet styles, HR's and tables are styled with preset "templates" (stylesheets) that can be selected from a drop down or icon based image gallery of styles. 

These templates like "marketing poster", or "event invitation" would stop users from adding their content to a PDF and uploading the PDF created in Word. (Because a Word offers a much better UX for creating content).
